packages/eyes/src/generate-snapshots/build-base-files.ts

```diff
@@ -0,0 +1,47 @@
+import Registry, {importMeta} from '@ui-autotools/registry';
+import {consoleLog} from '@ui-autotools/utils';
+import fs from 'fs';
+import path from 'path';
+import {generateFilename, generateData} from './filename-utils';
+
+export const buildBaseFiles = (projectPath: string) => {
+  consoleLog('Building base files...');
+  consoleLog('Importing Metafiles...');
+  importMeta();
+
+  const autotoolsFolder = path.join(projectPath, '.autotools');
+
+  if (!fs.existsSync(autotoolsFolder)) {
+    fs.mkdirSync(autotoolsFolder);
```

Comment:
What's the point of creating this directory, I don't see that we're using it later in this file.

Reply:
Using it now